src/style.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --star: url(./images/star.svg);
+  --active-star: url(./images/star-active.svg);
+}
+
+* {
+  margin: 0;
```

Comment:
https://github.com/mate-academy/layout_stars/pull/2514#discussion_r1109595108
in the another PR was comment about that :arrow_up: 
if you want to reset some default styles do it for a list of tags instead of `*` selector